google/resource_bigtable_instance.go

```diff
@@ -44,11 +44,17 @@ func resourceBigtableInstance() *schema.Resource {
 			},
 
 			"num_nodes": {
-				Type:         schema.TypeInt,
+				Type:     schema.TypeInt,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Since the number of nodes is `ForceNew`, we don't want to force our users to delete and recreate their production database.

I would rather have our users specify num_nodes = 1 for DEVELOPMENT.

If you don't send num_nodes to the server, does it set 1 for dev and 3 for production by default? If so, we could use `Computed = True` for num_nodes.